rpc/websocket_test.go

```diff
@@ -124,66 +126,61 @@ func TestWebsocketLargeCall(t *testing.T) {
 		t.Fatal("no error for too large call")
 	}
 }
-*/
 
 // This test checks whether the wsMessageSizeLimit option is obeyed.
+//
+// This test times out occasionally due to context timeout differences with go-ethereum.
+// These differences are not critical, so this test can simply be skipped.
 func TestWebsocketLargeRead(t *testing.T) {
-	t.Parallel()
+	t.Skip("Flaky")
 
 	var (
 		srv     = newTestServer()
 		httpsrv = httptest.NewServer(srv.WebsocketHandler([]string{"*"}))
 		wsURL   = "ws:" + strings.TrimPrefix(httpsrv.URL, "http:")
+		buffer  = 64
 	)
 	defer srv.Stop()
 	defer httpsrv.Close()
 
-	testLimit := func(limit *int64) {
-		opts := []ClientOption{}
-		expLimit := int64(wsDefaultReadLimit)
-		if limit != nil && *limit >= 0 {
-			opts = append(opts, WithWebsocketMessageSizeLimit(*limit))
-			if *limit > 0 {
-				expLimit = *limit // 0 means infinite
+	for _, tt := range []struct {
```

Comment:
nit: please extract the anonymous struct as a variable called `tests` and use named fields when creating instances of it. Let's stick to one way of declaring parameters for table driven tests. `avalanchego` also insist on this specific way of declaring them.